src/pages/index.js

```diff
@@ -28,49 +11,10 @@ export default function HomePage() {
 				<link rel="icon" href="/Dashy.webp" />
 			</Head>
 			<h1>Home</h1>
-			{loading && <div>Loading...</div>}
-			{error && <div>{error.message}</div>}
-			{data && (
-				<pre>
-					<code>{JSON.stringify(data, null, 4)}</code>
-				</pre>
-			)}
-			<section>
-				<Button
-					aria-label="decrement"
-					onClick={() => {
-						decrementCounter();
-					}}
-				>
-					-
-				</Button>
-				<input
-					value={`${counter}`}
-					size={2}
-					onChange={event => {
-						setCounter(Number.parseInt(event.target.value, 10));
-					}}
-				/>
-				<Button
-					aria-label="increment"
-					onClick={() => {
-						incrementCounter();
-					}}
-				>
-					+
-				</Button>
-			</section>
-			<br />
+
 			<section>
-				<Button
-					onClick={() => {
-						setId(nanoid());
-					}}
-				>
-					Generate ID
-				</Button>
 				<br />
```

Comment:
It is recommend to use CSS-Styling instead of `<br />` to get more space.

Reply:
Hi @stefankorder  , The index.js file was there as template. I didnt write it on my own :)

Reply:
But now you could edit file :)

Reply:
done